app/views/organizations/adopter_fosterer/dashboard/index.html.erb

```diff
@@ -3,6 +3,13 @@
   <% p.header_title t("dashboard.index.header_title") %>
   <% p.content do %>
     <div class="row">
+      <% unless current_user.adopter_foster_account.adopter_foster_profile.present? %>
+        <div class="alert alert-warning ">
+          <%= link_to t('.complete_my_profile'), new_adopter_fosterer_profile_path,
```

Comment:
@albertabney might you want this t (translation) to reference the en.yml?

if so, i think you'd want this to be:
``` <%= link_to t('organizations.adoptable_pets.show.complete_my_profile')```

Reply:
@maebeale Made the correction. Thanks